index.d.ts

```diff
@@ -0,0 +1,13 @@
+declare module "svelte-virtual-list" {
+  import { SvelteComponentTyped } from "svelte/types/runtime";
+
+  export interface IProps<T> {
+    items: T[];
+    height?: string;
+    itemHeight?: number;
+    start: number;
+    end: number;
+  }
+
+  export default class VirtualList<T> extends SvelteComponentTyped<IProps<T>> {}
```

Comment:
You should also add a type definition for the slots. That way you can type the generic relationship between `items` and `let:item`